src/sentry/uptime/config_producer.py

```diff
@@ -62,3 +73,53 @@ def _produce_to_kafka(
     )
     result = _configs_producer.produce(ArroyoTopic(topic), payload)
     result.result()
+
+
+def get_partition_from_subscription_id(subscription_id: UUID) -> int:
+    return int(subscription_id) % settings.UPTIME_CONFIG_PARTITIONS
+
+
+def get_partition_keys(subscription_id: UUID) -> tuple[str, str]:
+    partition = get_partition_from_subscription_id(subscription_id)
+    return f"uptime:configs:{partition}", f"uptime:updates:{partition}"
```

Comment:
nit: Sort of prefer this to be two functions, just so the naming is a bit more clear.

though you can pretty much tell from looking at the return strings, and at the call site what the unwrapped values are